packages/beeq/src/components/slider/bq-slider.tsx

```diff
@@ -281,6 +328,20 @@ export class BqSlider {
     );
   };
 
+  private renderTooltip = (value: number, thumbPosition: number): HTMLBqTooltipElement => (
+    <bq-tooltip
+      class="absolute"
+      exportparts="base,trigger,panel"
+      alwaysVisible={this.isTooltipAlwaysVisible}
+      visible
+      distance={this.enableValueIndicator ? 6 : 16}
+      style={{ left: `${thumbPosition}px`, fontVariant: 'tabular-nums' }}
+    >
+      <div class={{ 'absolute h-1 w-1': true, '-z-[1]': this.tooltipAlwaysVisible }} slot="trigger" />
+      {value}
```

Comment:
Take into account the decimal count, as per the label.

```suggestion
      {value.toFixed(this.decimalCount)}
```

![CleanShot 2024-04-12 at 18 20 30@2x](https://github.com/Endava/BEEQ/assets/328492/9f52624d-838f-477a-a30d-a1a6ce9d9d79)